src/MarginTrading.Backend.Services/AccountManager.cs

```diff
@@ -217,7 +228,7 @@ private IEnumerable<AccountStatsUpdateMessage> GenerateAccountsStatsUpdateMessag
             {
                 try
                 {
-                    var account = CreateAccount(clientId, baseAsset, tradingConditionsId);
+                    var account = await CreateAccountAndWalletAsync(clientId, baseAsset, tradingConditionsId);
```

Comment:
This method is used for both Live and Demo accounts. And we should not create wallets for Demo accounts.